src/client/components/StudyGroupLogo/StudyGroupLogo.styles.css

```diff
@@ -0,0 +1,16 @@
+i {
```

Comment:
Styles should only be applied to this component, do not write global styles.

Reply:
removed global styling and used classes instead